network-api/networkapi/wagtailpages/migrations/0013_auto_20220330_2304.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.12 on 2022-03-30 23:04
+
+from django.db import migrations, models
+import wagtail_color_panel.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0012_add_image_grid_block_to_blog'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='articlepage',
+            name='hero_background_color',
+            field=wagtail_color_panel.fields.ColorField(default='#ffffff', help_text='Background color of the hero section.', max_length=7),
+        ),
+        migrations.AddField(
+            model_name='articlepage',
+            name='hero_text_color',
+            field=models.CharField(choices=[('light', 'Light'), ('dark', 'Dark')], default='light', help_text='Color theme of hero section text. Choose light for light colored backgrounds and dark for darker color backgrounds.', max_length=25),
+        ),
+    ]
```

Comment:
**Suggestion:** I know, this has not been followed much in the project before, but I am certainly a fan of named migration files. You can create a named migration with `inv manage "makemigrations -n <description_of_the_migration>"`. It just helps to make sense of the different files in the migrations folder and to track history.

Reply:
Hi Tibor, thanks for the great suggestion, I have updated the migrations with the mentioned command to be descriptive about what the migration is doing!